### PR TITLE
Make some clarifications in the test plan

### PR DESCRIPTION
**Pull request by charisk**
Just some small things I noticed while going through the test plan.

## Checklist
N/A:
- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

**Review by shati-patel — APPROVED** [review]
Thanks!

File: docs/test-plan.md [hunk 1]
```diff
@@ -57,14 +57,16 @@ choose to go through some of the Optional Test Cases.
 2. Select the `google/brotli` database (or download it if you don't have one already)
 3. Run a local query.
 4. Once the query completes:
+   - Chose the `#select` result set from the drop-down
```
Comment (robertbrignull) [inline docs/test-plan.md:60]: I wonder if we want this when it's essentially working around a ~bug~ unintended feature. I suggested on the bug issue that maybe we don't want to use the `FunLinesOfCode` query in the test plan.

Though on balance it doesn't really hurt to add this step in, so I don't actually have a problem with it.

Comment (charisk) [reply docs/test-plan.md:60]: Let's continue the discussion on the bug issue. I'll get this in for now as it's an improvement. We can decide whether to change the behaviour and/or update the test plan there.

Comment (robertbrignull) [reply docs/test-plan.md:60]: As noted on the other issue, I'm not trying to block the PR. We can always change the testplan again if we want to.

**Pull request merged**